app/src/main/java/com/codesoom/assignment/interceptor/LoginCheckInterceptor.java

```diff
@@ -0,0 +1,53 @@
+package com.codesoom.assignment.interceptor;
+
+import com.codesoom.assignment.errors.UnauthorizedException;
+import com.codesoom.assignment.utils.JwtUtil;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpMethod;
+import org.springframework.util.StringUtils;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static org.springframework.util.StringUtils.*;
+
+@Slf4j
+@RequiredArgsConstructor
+public class LoginCheckInterceptor implements HandlerInterceptor {
+    private final JwtUtil jwtUtil;
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        String method = request.getMethod();
+
+        if (method.equals(HttpMethod.POST.name()) ||
+                method.equals(HttpMethod.PATCH.name()) ||
+                method.equals(HttpMethod.DELETE.name())
+        ) {
+            String authorization = request.getHeader("Authorization");
+
+            if (!hasText(authorization)) {
+                throw new UnauthorizedException("인증 정보가 없습니다. 인증 정보가 필요합니다.");
+            }
+
+            int prefixLength = "Bearer ".length();
+            String token = authorization.substring(prefixLength);
```

Comment:
이런건 final을 붙여줘도 괜찮겠죠.
```suggestion
            final int prefixLength = "Bearer ".length();
            final String token = authorization.substring(prefixLength);
```